rqd/rqd/rqcore.py

```diff
@@ -343,6 +339,16 @@ def runLinux(self):
 
         if rqd.rqconstants.RQD_PREPEND_TIMESTAMP:
             pipe_to_file(frameInfo.forkedCommand.stdout, frameInfo.forkedCommand.stderr, self.rqlog)
+        else:
+            with open(self.rqlog, 'a') as f:
```

Comment:
It looks like this broke file logging. `self.rqlog` is a file object and not a string.

Reply:
I'll take a look and fix that soon!

Thanks!

Reply:
I should not that #1401 changes alot of this logic anyway (if it gets merged)

Reply:
Hi @lithorus 

FYI ...

I changed the code to prevent crashes due to TypeError and fixed the broken file logging. Thanks!

This is the new PR: https://github.com/AcademySoftwareFoundation/OpenCue/pull/1417

Reply:
Hi @lithorus ,

Let me know when your PR https://github.com/AcademySoftwareFoundation/OpenCue/pull/1401 is ready for review. For now, it is on draft and some checks/pipelines are falling.

Looks like a nice feature! Thanks!

Reply:
Yes, I will write some examples and documentation on how it's used after #1416 is merged (some of the loki stuff is not compatible with python 2.x)
Also, seeing the recent sentry support in cuebot, it would be interesting to also add support for that aswell.

Reply:
Great. Thanks!